[38;5;127mlibs[0m/[38;5;172mnvidia-cuda[0m/[38;5;67m11.2.0[0m/[38;5;68mbin[0m
 |
 [0;32mOK[0m
ModuleCmd_Load.c(208):ERROR:105: Unable to locate a modulefile for '/users/adbt150'
/var/spool/slurmd.spool/job813044/slurm_script: line 21: /users/adbt150/yes/etc/profile.d/conda.sh: No such file or directory
usage: conda [-h] [-v] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'activate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'content-trust', 'doctor', 'repoquery', 'env')
Thu Mar  7 05:58:34 2024       
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 535.154.05             Driver Version: 535.154.05   CUDA Version: 12.2     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA A100-PCIE-40GB          Off | 00000000:8A:00.0 Off |                    0 |
| N/A   36C    P0              37W / 250W |      0MiB / 40960MiB |      0%      Default |
|                                         |                      |             Disabled |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
Traceback (most recent call last):
  File "/users/adbt150/lm-evaluation-harness/main.py", line 6, in <module>
    from lm_eval import tasks, evaluator, utils
  File "/users/adbt150/lm-evaluation-harness/lm_eval/tasks/__init__.py", line 4, in <module>
    import sacrebleu
ModuleNotFoundError: No module named 'sacrebleu'
Traceback (most recent call last):
  File "/users/adbt150/lm-evaluation-harness/main.py", line 6, in <module>
    from lm_eval import tasks, evaluator, utils
  File "/users/adbt150/lm-evaluation-harness/lm_eval/tasks/__init__.py", line 4, in <module>
    import sacrebleu
ModuleNotFoundError: No module named 'sacrebleu'
Traceback (most recent call last):
  File "/users/adbt150/lm-evaluation-harness/main.py", line 6, in <module>
    from lm_eval import tasks, evaluator, utils
  File "/users/adbt150/lm-evaluation-harness/lm_eval/tasks/__init__.py", line 4, in <module>
    import sacrebleu
ModuleNotFoundError: No module named 'sacrebleu'
